frontend/lib/api/polling_station/PollingStationListProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import { PollingStation, usePollingStationListRequest } from "@kiesraad/api";
+
+export interface iPollingStationListProviderContext {
+  pollingStations: PollingStation[];
+}
+
+export const PollingStationListProviderContext = React.createContext<iPollingStationListProviderContext | undefined>(
+  undefined,
+);
+
+export interface PollingStationListProviderProps {
+  children: React.ReactNode;
+  electionId: number;
+}
+
+export function PollingStationListProvider({ children, electionId }: PollingStationListProviderProps) {
+  const { data, loading } = usePollingStationListRequest({ election_id: electionId });
```

Comment:
Does this need to be in the global context / via a provider, instead of using 
usePollingStationListRequest() directly in the PollingStationListPage ?

Reply:
Was doubting about that but the Provider also handles the loading and error state (although both are not yet really implemented), so that the `PollingStationListPage` does not have to deal with that. Is that ok with you?

Reply:
I just worry about the pattern, will we add providers for all lists for example?

Reply:
Now that the empty state is specified, in the Page, there is really no excuse anymore for the Provider to exist :) I've removed it.